applications/nice1-tools-ecosystem.md

```diff
@@ -0,0 +1,278 @@
+
+
+## Development Roadmap
+
+### Overview
+Total Estimated Duration: 6 months
+
+Full-Time Equivalent (FTE): Two (2)
+
+Total Costs: $49,000 USD
+
+Note: Some of the milestones can possibly overlap and be done in parallel.
+
+## Milestone Summary
+
+
+### Milestone 1 - Nice1 Plugin Unity
+- Estimated duration: 2 months
+- FTE: 1
+- Costs: $8.300 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0  |
+| 0b. | Documentation | Documentation for Unity developers. |
+| 0c. | Unit tests | Unit tests for the plugin. |
+| 1. | Webview | Develop a webview plugin to call anchor-link functions. |  
+| 2. | C# Anchor-Link | Analyze the problem to develop a c# Anchor-link SDK. |  
+| 3. | Platform | Mac, Android and iOS version.|  
+
+### Milestone 2 - Nice1 Plugin Unreal
+- Estimated duration: 2 months
+- FTE: 1
+- Costs: $8.000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0  |
+| 0b. | Documentation | Documentation for Unreal developers |
+| 0c. | Unit tests | Unit tests for the plugin. |
+| 1. | Unreal Plugin | First plugin version running in Unreal.|    
+| 2. | Platform | Mac, Android and iOS version.|  
+
+
+### Milestone 3 - Nice1 Plugin Functionalities
+- Estimated duration: 4 months
+- FTE: 1
+- Costs: $9.700 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Connection and deserialization | Review the implementation as well as the current architecture to find a more optimal solution. |  
+| 2. | Legendary Legends Demo PC |Removal of dependency with third-party plugins, adaptation to the latest version of the plugin and bug fixes. |  
+| 3. | Extended EOS networks support | Telos, Proton, EOS, WAX, mainnet and testnet for all.|
+| 4. | Use of audited mechanics contracts | Manage from the plugin itself the contracts on different mechanics (licenses, character selection, etc).|  
+
+
+### Milestone 4 - Nice1 Link (Lvl1)
+- Estimated duration: 2 months
+- FTE: 1
+- Costs: $7.700 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0  |
+| 0b. | Documentation | Documentation for the frontend libraries for developers. |
+| 0c. | Unit tests | Unit tests for all the specified contracts, to ensure both proper functionality and better security. |
+| 1. | SC Profiles | Contract for members profiles management. |  
+| 2. | SC Licenses | Contract for managing product licenses sold by developers / product sellers. |  
+
+### Milestone 5 - Nice1 Link (Lvl2)
+- Estimated duration: 2 months
+- FTE: 1
+- Costs: $7.700 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | SC Staking | Contracts for staking both nfts and nice1 tokens and get tokens or nfts in exchange. |  
+| 2. | Nice JS tools | Javascript tools and components for easy nice1 integration in websites. |  
```

Comment:
Just clarifying here. These APIs will be Javascript and not Typescript which sometimes is thrown around interchangeably.

Reply:
Actually, the libraries will be built in typescript. But since the output is JS, we just simplified.